DEPRECATED_APIs.md

```diff
@@ -159,6 +159,7 @@
 | @aws-cdk/aws-ec2 | SubnetSelection.​subnetName | Use `subnetGroupName` instead |
 | @aws-cdk/aws-ec2 | SubnetType.​ISOLATED | use `SubnetType.PRIVATE_ISOLATED` |
 | @aws-cdk/aws-ec2 | SubnetType.​PRIVATE | use `PRIVATE_WITH_NAT` |
+| @aws-cdk/aws-ec2 | SubnetType.​PRIVATE_WITH_NAT | use `PRIVATE_WITH_EGRESS` |
```

Comment:
I don't think this is necessary. The first line of this package says `# List of deprecated APIs in v1`

Reply:
Removed this change